pkg/dns/dns_test.go

```diff
@@ -40,7 +41,7 @@ func genRandomBytes(size int) (blk []byte) {
 }
 
 func (s *DNSTestSuite) Test_DisabledServer() {
-	mockedDNSConsulClient := pbdns.NewMockDNSServiceClient(s.T())
```

Comment:
`NewMockDNSServiceClient` was removed from consul/proto-public, this moves the mock reference to the one generated in this repo